viewer/api/models.py

```diff
@@ -167,4 +167,7 @@ class Stats(models.Model):
     population = models.CharField(max_length=255, default='')
     subgroup = models.CharField(max_length=255, default='')
     key = models.CharField(max_length=100)
-    value = models.IntegerField()
+    value = models.FloatField(blank=True, null=True)
```

Comment:
Do we really want blank/null in `value`? I think if we don't have one, something has gone pretty wonky.

Reply:
This field is also overloaded for the general branch stats, like total pings and total clients so I was being lenient here. But now that you mention it those should also always have values.